app/news/newsItem.tsx

```
@@ -0,0 +1,19 @@
+import Link from "next/link";
+import styles from "./newsItem.module.css";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)